BREAKING.md

```diff
@@ -15,7 +15,14 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 -   Avoid using code formatting in the title (it's fine to use in the body).
 -   To explain the benefit of your change, use the [What's New](https://fluidframework.com/docs/updates/v1.0.0/) section on FluidFramework.com.
 
-# 2.0.0-internal.4.1.0
+## 2.0.0-internal.5.0.0 Upcoming changes
+
+-   [Upgraded Typescript target to ES2020](#Upgraded-Typescript-target-to-ES2020)
+
+### Upgraded Typescript target to ES2020
+
+Upgraded typescript transpilation target to ES2020. This is done in order to decrease the bundle sizes of Fluid Framework packages. This has provided size improvements across the board for ex. Loader, Driver, Runtime etc. Reduced bundle sizes helps to load lesser code in apps and hence also helps to improve the perf.
```

Comment:
i'm really curious what kind of gain you are seeing! please share that broadly at some point.

Reply:
At the time of creating the work item which was about 3-4 months ago, we estimated these savings. It would be arouud that:
<html>
<body>
<!--StartFragment-->

Area/Savings | Baseline | ES2018 savings | ES2018  %savings | ES2020 savings | ES2020  %savings
-- | -- | -- | -- | -- | --
Odsp Driver | 118.7 | 1.73 | 1.46% | 5.5 | 4.63%
Loader | 154 | 0.9 | 0.58% | 5 | 3.25%
Container Runtime | 200 | 2.6 | 1.30% | 8 | 4.00%
Total | 472.7 | 5.23 | 1.11% | 18.5 | 3.91%

<br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>